src/main/java/ai/rev/speechtotext/models/asynchronous/Summarization.java

```diff
@@ -0,0 +1,88 @@
+package ai.rev.speechtotext.models.asynchronous;
+
+import ai.rev.speechtotext.models.NlpModel;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * A Summarization Job object provides all the information associated with a summarization job
+ * submitted by the user.
+ */
+public class Summarization {
+  /** User defined prompt. * */
+  @SerializedName("prompt")
+  private String prompt;
+
+  /** Standard or Premium AI backend. * */
+  @SerializedName("model")
```

Comment:
not sure we can have any other model type, but we can say something like `summarization model` and then `@see NlpModel`. then we can extend the NlpModel as we need without other changes